INSTALL.md

```diff
@@ -267,13 +267,12 @@ Creating the database
 
         local   all    all    trust
 
-    By default, the password for the user musicbrainz should be "musicbrainz",
-    as stated in lib/DBDefs.pm. You can change it with `psql`:
-
-        postgres=# ALTER USER musicbrainz UNENCRYPTED PASSWORD 'musicbrainz'
+    You do not need to create this user in PostgreSQL yourself; the next step will
```

Comment:
I'd just clarify this part a bit since "this user" doesn't really follow from the previous paragraph
```suggestion
    You do not need to create the musicbrainz user in PostgreSQL yourself; the next step will
```

Reply:
Right, will tweak that tomorrow.

Reply:
Also noticed the dump import points directly to the main ftp (which doesn't even get turned into a hyperlink), instead of the server page that also lists the mirrors; will update that too.